src/Symfony/Component/DependencyInjection/composer.json

```diff
@@ -20,7 +20,7 @@
     },
     "require-dev": {
         "symfony/yaml": "~3.2",
-        "symfony/config": "~2.8|~3.0",
+        "symfony/config": "~3.3",
```

Comment:
should we add a conflict rule to avoid issues when getting a lower version for people using standalone components ?